azure-pipelines.yml

```diff
@@ -3,32 +3,50 @@
 # Add steps that build, run tests, deploy, and more:
```

Comment:
In spite of the fact that at least some of this file is pre-existing, it was never subjected to review. Feel free to comment on any part of it